packages/TypeDeclaration/tests/Rector/FunctionLike/ParamTypeDeclarationRector/FixtureUnionType/undesired.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\TypeDeclaration\Tests\Rector\ClassMethod\ParamTypeDeclarationRector\FixtureUnionType;
+
+/**
+ * @param real|int $a
+ * @param int $b
+ * @param int|void $c
+ */
+function someFunction($a, $b, $c) {
+}
+
+?>
+-----
+<?php
+
+namespace Rector\TypeDeclaration\Tests\Rector\ClassMethod\ParamTypeDeclarationRector\FixtureUnionType;
+
+/**
+ * @param real|int $a
+ * @param int $b
+ * @param int|void $c
+ */
+function someFunction(float|int $a, int $b, int|void $c) {
```

Comment:
@nikic Hi, is `int|void` legal for paramter? I didn't find it excluded in: https://github.com/nikic/PHP-Parser/commit/664c10121e9d3735ba958274f2ae737580303d9a
If not, what should be used here?